Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -243,8 +233,19 @@ public static void ApplyChanges(this InterceptorPool pool, IEnumerable<ITrackabl
                     context.ApplyChangesOnProperties(item, visitationHelper, TrackingState.Deleted, interceptors); 
                 }
             }
-        }
 
+        bool isModifiable(bool isComplex)
+        {
+          if (item.TrackingState == TrackingState.Modified
+              && (state == null || state == TrackingState.Modified)
+              && item.ModifiedProperties?.Count > 0)
```

Comment:
Can also be changed to:

    item.ModifiedProperties?.Any() == true

Essentially the same, just wasting another call doing it via LINQ.